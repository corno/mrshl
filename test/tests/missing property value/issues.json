[
    [ "missing property value", "error", 17, 1, 17, 2]
]