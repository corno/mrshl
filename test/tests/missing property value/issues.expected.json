[
    [ "deserialization", "warning", "property 'y' has default value, remove", 16, 5, 16, 8],
    [ "deserialization", "error", "missing property value", 17, 1, 17, 2]
]